tests/core/test_host.py

```diff
@@ -184,6 +184,14 @@ def test_stops_a_service(self, service):
         self.assertTrue(host.service_stop(service_name))
 
         service.assert_called_with('stop', service_name)
+    
```

Comment:
According to pep8 for py35 this line failed with:

```
tests/core/test_host.py:187:1: W293 blank line contains whitespace
```

Reply:
Thanks for the review. I have just pushed another commit fixing this issue.